### PR TITLE
[NFC][addkernels] Fix compiler warnings [-Wreserved-identifier]

### PR DESCRIPTION
**Pull request by averinevg**
The problem is described in https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1322

### CODE REVIEW

File: addkernels/addkernels.cpp [hunk 1]
```diff
@@ -104,7 +104,7 @@ void PrintHelp()
     std::cout << "           -n[o-recurse] : dont expand include files recursively. Default: off"
               << std::endl;
     std::cout << "           -m[ark-includes] : mark variables that represent include files with "
-                 "'__INC'. Default: off"
+                 "'_INC'. Default: off"
```
Comment (atamazov) [inline addkernels/addkernels.cpp:107]: [Recommendation] This is fine, but maybe this:
```suggestion
                 "'_INCLUDE'. Default: off"
```
would be cleaner? Up to you.

**Review by atamazov — APPROVED** [review]
LGTM

File: addkernels/addkernels.cpp (hunk 1 above)
Comment (averinevg) [reply addkernels/addkernels.cpp]: Changed as you suggested

**Review by atamazov — APPROVED** [review]
LGTM!

**Comment by atamazov** [comment]
This may block migration to 4.5 in our CI.

**Review by junliume — APPROVED** [review]

**Review by JehandadKhan — APPROVED** [review]

**Pull request merged**